packages/manager/src/features/CloudPulse/Widget/components/CloudPulseIntervalSelect.tsx

```diff
@@ -119,34 +120,35 @@ export const CloudPulseIntervalSelect = React.memo(
     );
 
     return (
-      <StyledWidgetAutocomplete
-        isOptionEqualToValue={(
-          option: IntervalOptions,
-          value: IntervalOptions
-        ) => {
-          return option?.value === value?.value && option?.unit === value?.unit;
-        }}
-        onChange={(
-          _: React.SyntheticEvent,
-          selectedInterval: IntervalOptions
-        ) => {
-          setSelectedInterval(selectedInterval);
-          onIntervalChange({
-            unit: selectedInterval?.unit,
-            value: selectedInterval?.value,
-          });
-        }}
-        textFieldProps={{
-          hideLabel: true,
-        }}
-        disableClearable
-        fullWidth={false}
-        label="Select an Interval"
-        noMarginTop={true}
-        options={[autoIntervalOption, ...availableIntervalOptions]}
-        sx={{ width: { xs: '100%' } }}
-        value={selectedInterval}
-      />
+      <CloudPulseTooltip title={'Data points aggregation'}>
```

Comment:
```suggestion
      <CloudPulseTooltip title={'Aggregation Interval'}>
```

or something like "Data point aggregation interval"

Reply:
This is as per the mock up. To change this, we need UX confirmation